drivers/serial/uart_stm32.c

```diff
@@ -1712,11 +1739,13 @@ STM32_UART_IRQ_HANDLER_DECL(index)					\
 									\
 PINCTRL_DT_INST_DEFINE(index);						\
 									\
+static const struct stm32_pclken pclken_##index[] =			\
+						   STM32_DT_INST_CLOCKS(index);\
+									\
 static const struct uart_stm32_config uart_stm32_cfg_##index = {	\
 	.usart = (USART_TypeDef *)DT_INST_REG_ADDR(index),		\
-	.pclken = { .bus = DT_INST_CLOCKS_CELL(index, bus),		\
-		    .enr = DT_INST_CLOCKS_CELL(index, bits)		\
-	},								\
+	.pclken = pclken_##index,						\
```

Comment:
Please fix indentation
```suggestion
	.pclken = pclken_##index,					\
```